### PR TITLE
INTYGFV-14282: Different text should be shown if all diagnoses are mi…

### PR DESCRIPTION
**Pull request by mhornfeldt**
…ssing fmb support

### CODE REVIEW

File: packages/webcert/src/components/fmb/FMBPanel.tsx [hunk 1]
```diff
@@ -43,6 +43,10 @@ const FMBPanel: React.FC = () => {
     }
   }
 
+  const hasNoFMBSupport = (): boolean => {
```
Comment (karlssone) [inline packages/webcert/src/components/fmb/FMBPanel.tsx:46]: Should it be fmbDiagnosisCodes.length > 1?

File: packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx [hunk 2]
```diff
@@ -23,14 +23,19 @@ const EmptyWrapper = styled.div`
...
+const FMBPanelDiagnosisInfo: React.FC<Props> = ({ fmbDiagnosisCodeInfo, hasNoFMBSupport }) => {
   if (!fmbDiagnosisCodeInfo.diagnosTitle) {
     return (
       <EmptyWrapper className="iu-m-none">
         <InfoBox type={'observe'}>
-          <p>För den angivna diagnosen finns för tillfället inget FMB-stöd.</p>
+          <p>
+            {hasNoFMBSupport
+              ? 'För de angivna diagnoserna finns för tillfället inget FMB-stöd.'
+              : 'För den angivna diagnosen finns för tillfället inget FMB-stöd.'}{' '}
```
Comment (karlssone) [inline packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx:37]: Should this be here? {' '}

Comment (karlssone) [inline packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx:37]: Is hasNoFMBSupport representing if there are more than one diagnose?

Comment (RobertDanielsson) [reply packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx:37]: I noticed that when formatting with prettier it adds these squiggly brackets with a string inside. Perhaps some rule that needs to be looked at

File: packages/webcert/src/components/fmb/FMBPanel.tsx [hunk 3]
```diff
@@ -43,6 +43,10 @@ const FMBPanel: React.FC = () => {
...
+    return fmbDiagnosisCodes.length > 1 && fmbDiagnosisCodes.find((value) => value.diagnosTitle) === undefined
```
Comment (karlssone) [inline packages/webcert/src/components/fmb/FMBPanel.tsx:47]: Should it be fmbDiagnosisCodes.length > 1?

File: packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx [hunk 4]
```diff
@@ -23,14 +23,19 @@ const EmptyWrapper = styled.div`
...
+const FMBPanelDiagnosisInfo: React.FC<Props> = ({ fmbDiagnosisCodeInfo, hasSeveralDiagnoses }) => {
   if (!fmbDiagnosisCodeInfo.diagnosTitle) {
     return (
       <EmptyWrapper className="iu-m-none">
         <InfoBox type={'observe'}>
-          <p>För den angivna diagnosen finns för tillfället inget FMB-stöd.</p>
+          <p>
+            {hasSeveralDiagnoses
+              ? 'För de angivna diagnoserna finns för tillfället inget FMB-stöd.'
+              : 'För den angivna diagnosen finns för tillfället inget FMB-stöd.'}{' '}
```
Comment (karlssone) [inline packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx:37]: Could this be removed?

**Review by karlssone — APPROVED** [review]

**Pull request merged**